viv_utils/__init__.py

```diff
@@ -114,60 +124,31 @@ def getWorkspace(fp, analyze=True, reanalyze=False, verbose=False, should_save=T
     return vw
 
 
-class LoggingObject(object):
```

Comment:
i think this object was ultimately a bad idea. well, it did work, but i question its performance, and more importantly, it discourages from using a single global logger like all the rest of our code.

Reply:
it was useful at one point, but yeah agreed!